Gemfile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
This line automatically makes rails treat every string in that file as a frozen string literal.